tests/xs/opacity_Fe_test_gamma_uns.py

```diff
@@ -0,0 +1,81 @@
+"""test for opacity calculation of metal lines
+
+  - This test calculates Fe opacity from VALD3 line list. (Comparison with petitRADTRANS opacity is shown in examples/comparisons/opacity_Fe_VALD3.ipynb)
+    The calculation of gamma is based on the classical approximation by Unsoeld (1955)
+    
+  Note: The input line list needs to be obtained from VALD3 (http://vald.astro.uu.se/). VALD data access is free but requires registration through the Contact form (http://vald.astro.uu.se/~vald/php/vald.php?docpage=contact.html). After the registration, you can login and choose the "Extract Element" mode.
+        For this test, the request form should be filled as:
+            Starting wavelength :    1500
+            Ending wavelength :    100000
+            Element [ + ionization ] :    Fe 1
+            Extraction format :    Long format
+            Retrieve data via :    FTP
+            Linelist configuration :    Default
+            Unit selection:    Energy unit: eV - Medium: vacuum - Wavelength unit: angstrom - VdW syntax: default
+        Please rename the file sent by VALD ([user_name_at_VALD].[request_number_at_VALD].gz) to "vald2600.gz" if you would like to use the code below without editing it.
+   
+"""
+
+import pytest
+import numpy as np
+from exojax.spec import xsection, moldb, atomll
+from exojax.spec.hitran import SijT, doppler_sigma
+import matplotlib.pyplot as plt
+from exojax.utils.constants import m_u
+
+filepath_VALD3 = '/home/tako/work/VALD3/vald2600.gz'
```

Comment:
Can you change this path to .database?

Reply:
Note that exojax does not assume a common path to the database. If not specified exojax searches for the database in the local .database directory.